components/movementsensor/adxl345/adxl345.go

```diff
@@ -291,10 +507,33 @@ func (adxl *adxl345) Accuracy(ctx context.Context, extra map[string]interface{})
 	return map[string]float32{}, movementsensor.ErrMethodUnimplementedAccuracy
 }
 
+func contains(s []string, str string) bool {
```

Comment:
Yes, I also did a brief search and was surprised to not find one. What do you mean by "it's not very DRY"

Reply:
DRY is "Don't Repeat Yourself." The idea is that if you want the same functionality in many places, make a helper function and call that from many places. That way, if there's a bug to fix or a feature to add, you just need to change the helper function once, and you can piggyback on previous work instead of reimplementing `contains` every few months in another file. 

I'm surprised that the Go community doesn't nudge people towards this.